content/posts/android-tv-13.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Android TV v0.13
+subtitle: That was quick
+author: Niels van Velzen
+githubusername: nielsvanvelzen
+date: 2022-03-14
+justify: center
+---
+
+<!-- markdownlint-disable MD033 MD036 -->
+
+We're back with another release for Android TV and Fire TV users containing a lot of bugfixes!
+
+<!--more-->
+
+As promised in the previous release post, updates are getting quicker. No more waiting for 2 years anymore. Today's update brings some new features and greatly improves playback of videos. Let's get started with the player changes.
```

Comment:
> No more waiting for 2 years anymore.

This reads a little redundant.

Reply:
maybe something like: "As promised in the previous release post, the time between updates is getting shorter. We hope future updates will happen significantly more frequently than the two years between 0.11 and 0.12"